yt_dlp/extractor/vimeo.py

```diff
@@ -121,7 +122,13 @@ def _parse_config(self, config, video_id):
         video_data = config['video']
         video_title = video_data.get('title')
         live_event = video_data.get('live_event') or {}
-        is_live = live_event.get('status') == 'started'
+        live_status = traverse_obj(live_event, ('status', {lambda status: {
+            'pending': 'is_upcoming',
+            'active': 'is_live',  # TODO: is live or not?
+            'started': 'is_live',
+            'ended': 'post_live',
+        }.get(status)}))
```

Comment:
```suggestion
        live_status = {
            'pending': 'is_upcoming',
            'active': 'is_live',  # TODO: is live or not?
            'started': 'is_live',
            'ended': 'post_live',
        }.get(live_event.get('status'))
```
unnecessarily complex